src/lib/isEthereumAddress.js

```diff
@@ -0,0 +1,8 @@
+import assertString from './util/assertString';
+
+const eth = /^(0x)[0-9a-f]{40}$/i;
```

Comment:
I am just asking, don't we have any capitalize/uppercase for `Ethereum Address`?

Reply:
Ethereum addresses can be either all-caps or all-lower, which is why I added the `/i` flag to mark the regex as case insensitive. Addresses with mixed casing (e.g. `0xdAC17F958D2ee523a2206206994597C13D831ec7`) contain checksum information to make sure it's correct, but they require additional dependencies to compute and check.